Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -236,11 +255,16 @@ static u32  g_ErrorCode = 0;
 static bool g_bDiscInside = false;
 bool g_bStream = false;
 static bool g_bStopAtTrackEnd = false;
-static int  tc = 0;
+static int  finish_execute_command = 0;
+static int  finish_execute_read_command = 0;
 static int  dtk = 0;
 
 static u64 g_last_read_offset;
 static u64 g_last_read_time;
+static u64 g_last_command_time;
+
+static std::deque<DVDReadCommand> read_commands_to_execute;
+static u64 next_read_command_id;
```

Comment:
Why do we have a queue here?  As far as I know, it isn't possible to send multiple read commands to the DVD drive at the same time.  (It's possible WII_IPC_HLE_Device_DI needs a queue, but that shouldn't be here.)